src/Pos/Client/Txp/History.hs

```diff
@@ -92,13 +92,16 @@ getSenders UnsafeTx {..} = do
 data TxHistoryEntry = THEntry
     { _thTxId        :: !TxId
     , _thTx          :: !Tx
-    , _thInputs      :: ![TxOut]
     , _thDifficulty  :: !(Maybe ChainDifficulty)
-    , _thInputAddrs  :: ![Address]  -- TODO: remove in favor of _thInputs
```

Comment:
is frontend UI using this field ? If yes, we should notify them how to get Addresses without that field @Martoon-00